src/getfoods.js

```diff
@@ -0,0 +1,36 @@
+const displayData = (arr) => {
+  const board = document.querySelector('.container-food-cards');
+  board.innerHTML = '';
+  arr.forEach((food) => {
+    const item = document.createElement('article');
+    item.classList.add('card-food');
+    item.innerHTML = `
+          <div class='card-title'>
+            <h5>${food.strMeal}</h5>
+          </div>
+          <div class=btn-recipe>
+            <button type='button'>Recipe</button>
+          </div>
+          <div class='btn-liked'>
+            <button type='button'>Like</button>
+          </div>
+          <div class='card-img'>
+           <img src="${food.strMealThumb}" class='img-food'>        
+          </div>     
+      `;
+    item.id = food.idMeal;
+    // console.log(food);
```

Comment:
- [ ] _[OPTIONAL]_ Kindly don't forget to remove all the `console.log();` in your code :octopus:

Reply:
Thank you for this @IngDavidTM .
I'll make the necessary change suggestions.